src/modules/components/proposal/proposalDataListItem/proposalDataListItemStructure/proposalDataListItemStructure.test.tsx

```diff
@@ -125,7 +126,8 @@ describe('<ProposalDataListItemStructure/> component', () => {
         expect(screen.getByText(testProps.title)).toBeInTheDocument();
         expect(screen.getByText(testProps.summary)).toBeInTheDocument();
         expect(screen.getByText(testProps.status)).toBeInTheDocument();
-        expect(screen.getByText(testProps.date)).toBeInTheDocument();
+        const formattedDate = '5 hours left';
+        expect(screen.getByText(formattedDate)).toBeInTheDocument();
```

Comment:
Could you move the date test to a separate test and cover both the `left` and `ago` cases?

Reply:
✅